tests/test_job.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pybuildkite.jobs import Jobs
+from pybuildkite.client import Client
+from requests.exceptions import HTTPError
+
+class TestJob:
```

Comment:
There is already a TestJob class in `test_jobs.py` I suggest just adding these methods to that one =)